android/app/src/main/java/org/fossasia/openevent/utils/FilterableRealmLiveData.java

```diff
@@ -0,0 +1,44 @@
+package org.fossasia.openevent.utils;
+
+import android.arch.lifecycle.MutableLiveData;
+
+import java.util.List;
+
+import io.reactivex.Observable;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.functions.Predicate;
+import timber.log.Timber;
+
+/**
+ * A wrapper around MutableLiveData
+ * Allows filtering of the LiveData components using ReactiveX predicate.
+ * Should only be used with LiveData wrapping a List.
+ * @param <V>: The data type of the List.
+ */
+public class FilterableRealmLiveData<V> extends MutableLiveData<List<V>> {
+
+    private List<V> unfilteredData;
+    private CompositeDisposable compositeDisposable;
+
+    public FilterableRealmLiveData(List<V> unfilteredData) {
```

Comment:
Hmm, this should just be like other RealmDataWrappers, it should also handle the realm stuff, like other wrappers. It is just an extension for those guys

This is not handling the registering and unregistering of data 

Please correct me if I'm wrong. Can I use this as a drop in replacement for non filtering wrappers?

Reply:
>  Can I use this as a drop in replacement for non filtering wrappers?

I don't think so, I have only handled the filter functionality for the MutableLiveData, everything else is same as the MutableLiveData, hence it doesn't handle the realm stuff. It is an extension for MutableLiveData not a Realm wrapper.

Reply:
It should be just like wrapper with this extra functionality